### ISSUE TITLE
Ubuntu ARM Portability: hwcap.h not available.

### ISSUE THREAD

**Issue by myungjoo** [nnstreamer/nnstreamer#2704]
Ubuntu Bionic ARM64 build log:

```
FAILED: gst/nnstreamer/73d9e45@@nnstreamer@sha/tensor_filter_common.c.o 
cc -Igst/nnstreamer/73d9e45@@nnstreamer@sha -Igst/nnstreamer -I../gst/nnstreamer -Igst/nnstreamer/./include/ -I../gst/nnstreamer/./include/ -Igst/nnstreamer/tensor_transform -I/usr/include/glib-2.0 -I/usr/lib/aarch64-linux-gnu/glib-2.0/include -I/usr/include/gstreamer-1.0 -I/usr/include/orc-0.4 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Werror -std=gnu89 '-DVERSION="1.5.3"' '-DVERSION_MAJOR="1"' '-DVERSION_MINOR="5"' '-DVERSION_MICRO="3"' -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations -Wmissing-include-dirs -Wmissing-prototypes -Wnested-externs -Waggregate-return -Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="/etc/nnstreamer.ini"' -DENABLE_TENSORFLOW_LITE=1 -DENABLE_PYTORCH=1 -DENABLE_CAFFE2=1 -DHAVE_ORC=1 -DENABLE_FLATBUF=1 -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -pthread -MD -MQ 'gst/nnstreamer/73d9e45@@nnstreamer@sha/tensor_filter_common.c.o' -MF 'gst/nnstreamer/73d9e45@@nnstreamer@sha/tensor_filter_common.c.o.d' -o 'gst/nnstreamer/73d9e45@@nnstreamer@sha/tensor_filter_common.c.o' -c /<<PKGBUILDDIR>>/gst/nnstreamer/tensor_filter/tensor_filter_common.c
/<<PKGBUILDDIR>>/gst/nnstreamer/tensor_filter/tensor_filter_common.c:32:10: fatal error: asm-arm/hwcap.h: No such file or directory
 #include <asm-arm/hwcap.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

This codes appears to be not portable for Ubuntu.

### This issue is NOT RELEASE-CRITICAL; thus, its fix will not be merged before 1.6.0 release.

**Comment by tony-jinwoo-ahn** [comment]
I meet same error on rpi4b, ubuntu18.04 arm64.
I reverted a patch to fix build error:
```
commit ae2cac294aa8228c68523499e4690ae274169695
[ext/nnfw] Check availability of neon at runtime
```

**Comment by kparichay** [comment]
@tony-jinwoo-ahn Yes, as you said, `ae2cac294aa8228c68523499e4690ae274169695` commit causes the issue with the latest build. This will be fixed once #2699 merges.

**Comment by wooksong** [comment]
@tony-jinwoo-ahn I confirmed that this issue is resolved after merging #2699.
